docs/whatsnew.rst

```diff
@@ -93,82 +93,56 @@ space's name wiki".
 Technical novelties in eXo Platform 5.1
 ========================================
 
-**Upgrades**
+**New features**
 
-In the version 5.0, many tools/applications used by eXo Platform were 
-upgraded to newer  versions in order to benefit from their latest 
-updates and improvements: 
+In the version 5.1, we integrated a new feature which is the push notifications on mible.
+This new feature is supported by eXo mobile applications. It relies on Firebase Cloud Messaging 
+and a new addon packaged by default in the Enterprise Edition (exo-push-notifications).
 
-- Tomcat 8.5
+**Upgardes**
 
-- JBoss EAP 7.0
+In 5.1 version, eXo Platform upgraded to Jboss EAP 7.1 in order to:
 
-- Elasticseach 5.6 
+- Benefit from its lastest improvements such as:
 
-- Infinispan 8
+  - Use the new security framework WildFly Elytron
 
-- Groovy 2.4
+  - Support for HTTP/2
```

Comment:
No need to list JBoss EAP improvements IMO. We can just add a link to https://www.redhat.com/en/blog/red-hat-releases-jboss-eap-71 .